integration/readme.md

```diff
@@ -0,0 +1,15 @@
+#  Integration tests
+
+These tests run a gqlgen server against the apollo client to test real world connectivity.
+
+First start the go server
+```bash
+go run integration/server/server.go
+```
+
+And in another terminal:
+```bash
+cd integration
+npm install
```

Comment:
```suggestion
npm ci
```
https://docs.npmjs.com/cli/ci
npm ci is better than install.
it doesn't update package-lock.json unexpectedly.